beater/test_approved_es_documents/TestPublishIntegrationSpans.approved.json

```diff
@@ -327,7 +327,7 @@
                         "context": {
                             "post": [
                                 "    ins.currentTransaction = prev",
-                                "}"
+                                "{}"
```

Comment:
I know, but is there any problem with this?

Reply:
If it _needs_ to be changed for the script to work, then it is adding a limitation which input data can be tested. That could be considered a problem, at least it should be some conscious decision to add this limitation.

Reply:
It is conscious, yes.

Reply:
It was not obvious for me, please add a note to the PR description next time you introduce such a limitation - it should be obvious for everyone seeing this PR, especially the reviewer, as this can easily be overlooked. 

I am not convinced introducing this limitation is a good thing to do.

Reply:
> I am not convinced introducing this limitation is a good thing to do.

You don't mention any tangible, existing, issue; but I'll close this then. Will figure out some other way of handling this.

Reply:
I did not imply you should close this, on the contrary I offered to have an offline discussion to clarify. One of the purposes of a review is to point out if a code change introduces something concerning, which is what I did here, since this change limits which data can be used in tests. It is not a huge deal, as it only impacts the tests but it needs to be mentioned that this change is introduced.

Reply:
Sure. This was a time-boxed effort, thou. When I find some spare time, I will look into alternatives.